index.html

```diff
@@ -316,11 +316,11 @@ <h2>Contact me</h2>
           </p>
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- Add a license to your project.

Reply:
Okay I'll